KoreBuild-dotnet/build/KoreBuild.cmd

```diff
@@ -9,6 +9,9 @@ IF "%NUGET_PATH%"=="" (
 	ECHO Error: NUGET_PATH is not set.
 	EXIT /B 1
 )
+IF "%CHANNEL%"=="" (
```

Comment:
Yeah, I can. Though since the scripts read the environment variable, it won't change the fact that `CHANNEL` will work :)